tasks.py

```diff
@@ -141,13 +141,18 @@ def _get_latest_version() -> Version:
 
 def _bump_version(version_: Version, pre_release: bool) -> Version:
     if not pre_release:
-        # standard release
-        new_version = Version(f"{version_.major}.{version_.minor}.{version_.micro + 1}")
+        # standard release - remove the dev component if it exists
+        if version_.dev:
+            new_version = Version(f"{version_.major}.{version_.minor}.{version_.micro}")
```

Comment:
Is this var still in scope when it is referenced below since it is not first available outside the if/else statements. And did you test this locally first?

Reply:
It's not in scope in the else blocks but each block is creating initializing a `new_version` object.
`mypy` would catch it if I was forgetting a case and not returning a `Version` object.

Yes I've tested it locally.

The reason I'm changing it isn't because the function didn't work but because it was doing the wrong thing.
For example `0.0.3dev1` comes before `0.0.3`, not after it.